typescript/web/src/components/labeling-tool/image-navigation/__tests__/image-navigation-tool.tsx

```diff
@@ -1,151 +1,86 @@
 });
 
 test("should display one when only one image in list", async () => {
-  const imageId = await createImage("testImage");
+  // const imageId = await createImage("testImage");
   (useRouter as jest.Mock).mockImplementation(() => ({
-    query: { imageId, datasetSlug: "test-dataset", workspaceSlug: "local" },
+    query: {
+      imageId: BASIC_IMAGE_DATA.id,
+      datasetSlug: BASIC_IMAGE_DATA.dataset.slug,
+      workspaceSlug: BASIC_IMAGE_DATA.dataset.workspace.slug,
+    },
   }));
-
   renderImageNavigationTool();
-
   await waitFor(() =>
     expect(screen.queryByDisplayValue(/1/i)).toBeInTheDocument()
   );
   await waitFor(() => expect(screen.queryByText(/1/i)).toBeInTheDocument());
 });
 
-test("should select previous image when the left arrow is pressed", async () => {
+const testNextPrevImage = (isTestNext: boolean) => async () => {
   const mockedPush = jest.fn();
-  const oldestImageId = await createImage("testImageA");
-  incrementMockedDate(1);
-  const imageId = await createImage("testImageB");
-  incrementMockedDate(1);
-
-  await createImage("testImageC");
   (useRouter as jest.Mock).mockImplementation(() => ({
-    query: { imageId, datasetSlug: "test-dataset", workspaceSlug: "local" },
+    query: {
+      imageId: DEEP_DATASET_WITH_IMAGES_DATA.images[1].id,
+      datasetSlug: DEEP_DATASET_WITH_IMAGES_DATA.slug,
+      workspaceSlug: DEEP_DATASET_WITH_IMAGES_DATA.workspace.slug,
+    },
     push: mockedPush,
   }));
-
   const { container } = renderImageNavigationTool();
-
-  // We need to make sure that images have been loaded
   await waitFor(() =>
     expect(
-      screen.getByRole("button", { name: /^Previous image$/i })
+      screen.getByRole("button", {
+        name: isTestNext ? /^Next image$/i : /^Previous image$/i,
+      })
     ).toBeDefined()
   );
-
-  userEvent.type(container, "{arrowleft}");
-
+  userEvent.type(container, isTestNext ? "{arrowright}" : "{arrowleft}");
   expect(mockedPush).toHaveBeenCalledWith(
-    `/local/datasets/test-dataset/images/${oldestImageId}`
-  );
-});
-
-test("should select next image when the right arrow is pressed", async () => {
-  const mockedPush = jest.fn();
-  await createImage("testImageA");
-  incrementMockedDate(1);
-  const imageId = await createImage("testImageB");
-  incrementMockedDate(1);
-  const newestImageId = await createImage("testImageC");
-
-  (useRouter as jest.Mock).mockImplementation(() => ({
-    query: { imageId, datasetSlug: "test-dataset", workspaceSlug: "local" },
-    push: mockedPush,
-  }));
-  const { container } = renderImageNavigationTool();
-
-  // We need to make sure that images have been loaded
-  await waitFor(() =>
-    expect(screen.getByRole("button", { name: /^Next image$/i })).toBeDefined()
+    `/${DEEP_DATASET_WITH_IMAGES_DATA.workspace.slug}/datasets/${
+      DEEP_DATASET_WITH_IMAGES_DATA.slug
+    }/images/${DEEP_DATASET_WITH_IMAGES_DATA.images[isTestNext ? 2 : 0].id}`
```

Comment:
Can we move it into a function or at-least use intermediary variables to simplify the interpolation line?